openfpga/src/base/openfpga_gen_repack_constraints_template.h

```diff
@@ -0,0 +1,127 @@
+#pragma once
+
+#include "command.h"
+#include "command_context.h"
+#include "command_exit_codes.h"
+#include "openfpga_port.h"
+#include "openfpga_port_parser.h"
+#include "pcf_reader.h"
+#include "repack_design_constraints.h"
+#include "vtr_log.h"
+#include "write_xml_repack_design_constraints.h"
+#include "globals.h"
+
+/* begin namespace openfpga */
+namespace openfpga {
+
+static inline RepackDesignConstraints gen_repack_constraints(openfpga::PcfData* pcf_data){
+  auto& cluster_ctx = g_vpr_ctx.clustering();
```

Comment:
Avoid to use any more global variables. Pass them through arguments (const reference). We have seen enough troubles by using global variables. Any new codes should be clean at the beginning.